projects/jakarta-mail-api/project.yaml

```diff
@@ -0,0 +1,14 @@
+homepage: "https://eclipse-ee4j.github.io/mail"
+language: jvm
+primary_contact: ""
```

Comment:
@oliverchang do we have a preference for not including this field vs leaving it blank?